src/main/java/org/spongepowered/common/config/category/CustomDataRegistrationCategory.java

```diff
@@ -30,72 +30,68 @@
-                                                + "registered custom data manipulators by plugins. Since \n"
-                                                + "the list is generated AFTER the game starts, modifying \n"
-                                                + "this list will not affect Sponge's system in any way. \n"
-                                                + "However, it is advisable to view what registered datas \n"
-                                                + "exist on a server instance, such that when Sponge completes \n"
-                                                + "startup, it will be verified that all existing registrations \n"
-                                                + "are accounted for. A warning will be emitted for any existing \n"
-                                                + "registrations that were not registered, and moved to the \n"
-                                                + "'failed-data-list'.")
+    @Setting(value = "registered-data", comment = ""
+            + "An auto generated list, by Sponge, to provide a list of\n"
+            + "registered custom data manipulators by plugins. Since\n"
+            + "the list is generated AFTER the game starts, modifying\n"
+            + "this list will not affect Sponge's system in any way.\n"
+            + "However, it is advisable to view what registered datas\n"
+            + "exist on a server instance, such that when Sponge completes\n"
+            + "startup, it will be verified that all existing registrations\n"
+            + "are accounted for. A warning will be emitted for any existing\n"
+            + "registrations that were not registered, and moved to the\n"
+            + "'failed-data-list'.")
     private Set<String> registeredDataIds = new ConcurrentSkipListSet<>();
 
     /**
-     * Again, another auto-populated list of FAILED id's. This is never
-     * read from file, except after startup, to specify which id's can
-     * be "saved".
+     * Again, another auto-populated list of FAILED id's. This is never read from
+     * file, except after startup, to specify which id's can be "saved".
      */
-    @Setting(value = "failed-data-list", comment = "An auto generated list, by Sponge, to discover and list \n"
-                                                 + "all failed custom data deserializations at runtime due \n"
-                                                 + "to a lack of the registrations being made by a plugin. \n"
-                                                 + "Not to be confused by failed deserialization due to bad data. \n"
-                                                 + "Modifying the list will result in no effect as Sponge auto \n"
-                                                 + "generates this list. This is merely for user configuration.")
+    @Setting(value = "failed-data-list", comment = ""
```

Comment:
This might seem like a nitpick, but there should not be an empty string like this at the end of a line. I would much rather just put `comment = ` on the next line (though, what was wrong with what was there before?)

Reply:
I changed it to this variant to put more characters in each line. So there are roughly 100 characters in each line both in the yaml and docs.
If I move comment to a new line then I only have 80 chars.
Also in most cases, the 100 char limit seem to match the sentence part length, so the lines end with a comma or dot.